test/features/can_refresh_a_closed_record_test.rb

```diff
@@ -0,0 +1,233 @@
+            {"Type":"Email","Value":"podaac@podaac.jpl.nasa.gov"}]},"FirstName":"User","LastName":"Services",
+            "Roles":["Investigator","Technical Contact"]}],"LongName":"Physical Oceanography Distributed Active Archive Center, 
+            Jet Propulsion Laboratory, NASA","Roles":["ARCHIVER"],"ShortName":"NASA/JPL/PODAAC"}],"TemporalKeywords":["1 Day"],
+            "Platforms":[{"Characteristics":[{"DataType":"FLOAT","Description":"Nominal period of a spacecraft`s single revolution on an orbital plane.",
+            "Name":"OrbitPeriod","Unit":"Minutes","Value":"0.0"},{"DataType":"FLOAT",
+            "Description":"Spacecraft angular distance from orbital plane relative to the Equator.",
+            "Name":"InclinationAngle","Unit":"Degrees","Value":"0.0"}],
+            "Instruments":[{"Characteristics":[{"DataType":"FLOAT",
+            "Description":"Spacecraft angular distance from orbital plane relative to the Equator.",
+            "Name":"SwathWidth","Unit":"Kilometers","Value":"0.001"}],"LongName":"Conductivity, Temperature, Depth",
+            "ShortName":"CTD"},{"Characteristics":[{"DataType":"FLOAT",
+            "Description":"Spacecraft angular distance from orbital plane relative to the Equator.",
+            "Name":"SwathWidth","Unit":"Kilometers","Value":"0.001"}],"LongName":"Meteorological Measurement System",
+            "ShortName":"MMS"},{"Characteristics":[{"DataType":"FLOAT",
+            "Description":"Spacecraft angular distance from orbital plane relative to the Equator.","Name":"SwathWidth","Unit":"Kilometers",
+            "Value":"0.001"}],"LongName":"Current Meter","ShortName":"CURRENT METERS"}],"LongName":"SPURS-I WHOI Waveglider",
+            "ShortName":"Waveglider","Type":"In Situ Ocean-based Platforms"}],
+            "ArchiveAndDistributionInformation":{"FileDistributionInformation":[{"Format":"NETCDF","FormatType":"Native"}]}}}]}',
+            headers: {}
+        )
+
+    # stub_request(:get, "#{Cmr.get_cmr_base_url}/search/granules.echo10?revision_id=4")
+    #     .with(
+    #         headers: {'Accept-Encoding' => 'gzip;q=1.0,deflate;q=0.6,identity;q=0.3', 'Accept' => '*/*', 'User-Agent' => 'Ruby'}
+    #     )
+    #     .to_return(status: 200, body: '<?xml version="1.0" encoding="UTF-8"?><results><hits>0</hits><took>32</took></results>', headers: {})
+  end
+
+  describe "GET #refresh" do
+    it "refreshes a closed record and notifies of no new revision" do
+      visit '/home'
+      find("#closed_records_button").click
+      within '#closed' do
+        all('#record_id_')[0].click
+      end
+      sleep 1
+      screenshot_and_open_image
+      click_on 'Refresh'
+      # assert has_content? 'Latest revision for Collection has already been ingested '
+      page.must_have_content(`Latest revision for Collection has already been ingested`)
+    end
+
+    it "refreshes a closed record and notifies that a new revision is available to be ingested" do
```

Comment:
I think you want this second test to say, "refreshes a record and informs user a new revision is has been ingested."    I'll slack you the details of how this should work.